packages/runtime/src/state/react-builder-preview.ts

```diff
@@ -569,11 +570,6 @@ export function messageChannelMiddleware(
             window.getSelection()?.removeAllRanges()
             break
 
-          case ActionTypes.SET_LOCALIZED_RESOURCE_ID: {
-            client.setLocalizedResourceId(action.payload)
-            break
```

Comment:
Moved `SET_LOCALIZED_RESOURCE_ID` logic here: https://github.com/makeswift/makeswift/pull/836/files#diff-77b56149aa218322fa2db5bbccfd0e1e813309d76ea228b2790afb409dc3ead7R34

Reply:
`makeswiftApiClientSyncMiddleware` ensures that the action gets forwarded to the API client store.